### PR TITLE
update changelog for v1.0.0

### PR DESCRIPTION
**Pull request by payno**
Here are the change logs for 1.0. Please have a look and if you see anything missing or that could be ignore let me know :)

### CODE REVIEW

**Review by woutdenolf — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,133 @@
...
 ==========
 
+1.0.0: 2021/11/XX
+-----------------
+
+This the first version of `silx` supporting `PySide6` (for `Qt6`) and using `pytest` to run the tests.
+
+* `silx view`:
+
+  * Added Windows installer generation (PR #3548)
```
Comment (mretegan) [inline CHANGELOG.rst:11]: This was already present before this version, so maybe "updated" is more appropriate.

Comment (t20100) [reply CHANGELOG.rst:11]: There was only a .zip file, no proper installer... even though the fat binary generation was already there (but I couldn't found its addition in the changelog).

Comment (mretegan) [reply CHANGELOG.rst:11]: Ok, sounds good to me then.

**Review by t20100 — APPROVED** [review]

**Review by mretegan — APPROVED** [review]

**Review by vasole — APPROVED** [review]

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,134 @@
...
+        * Fixed issue with profile window size (PR #3455)
+
+      * `silx.gui.plot.PlotWidget`:
+
+        * Fixed update of `Scatter` item binned statistics visualization (PR #3452)
+        * Fixed OpenGL backend memory leak (PR #3453)
+        * Enhanced: Optimized scatter when rendered as regular grid with the OpenGL backend (PR #3447)
+        * Enhanced axis limits management by the OpenGL backend (PR #3504)
+        * Enhanced control of repaint (PR #3449)
+	* Enhanced text label background rendering with OpenGL backend (PR #3565)
```
Comment (t20100) [inline CHANGELOG.rst:70]: Added PR #3565 to changelog

**Pull request merged**